### ISSUE TITLE
[Bug]: Character direction does not change during fast animation

### ISSUE THREAD

**Issue by tkshnwesper** [RafaelBarbosatec/bonfire#523]
### What happened?

In versions 3.9.2 onward, fast animation is not affected by the character's direction. For instance, the shooting animation in the clip below does not flip if the character turns the other way.

https://github.com/RafaelBarbosatec/bonfire/assets/7362366/ae598fbb-3ff9-4dcb-b89b-968bcb60e35f

However, in version 3.9.0, fast animation changed with direction as shown below.

https://github.com/RafaelBarbosatec/bonfire/assets/7362366/d1e89311-5a54-49ae-8f4e-15d9c0e3f664

### Steps to reproduce?

```Markdown
1. Try using `animation?.playOnce` or `animation?.playOnceOther` API to reproduce this issue.
```

### What did you expect to happen?

Animation should flip based on direction.

### Bonfire version

3.9.3

**Comment by tkshnwesper** [comment]
Sent a PR with a fix #524

**Comment by RafaelBarbosatec** [comment]
Available in the version `3.9.4`
To this behavior set `useCompFlip` = true.
